src/components/authorization/roles/experience-operations-global.role.ts

```diff
@@ -0,0 +1,473 @@
+import { Sensitivity } from '../../../common';
+import { DbBudget } from '../../budget/model';
+import { DbBudgetRecord } from '../../budget/model/budget-record.model.db';
+import { DbCeremony } from '../../ceremony/model';
+import { DbInternshipEngagement, DbLanguageEngagement } from '../../engagement/model';
+import { DbEthnoArt } from '../../ethno-art/model';
+import { DbFieldRegion } from '../../field-region/model';
+import { DbFieldZone } from '../../field-zone/model';
+import { DbDirectory, DbFile } from '../../file/model';
+import { DbFileVersion } from '../../file/model/file-version.model.db';
+import { DbFilm } from '../../film/model';
+import { DbFundingAccount } from '../../funding-account/model';
+import { DbEthnologueLanguage, DbLanguage } from '../../language/model';
+import { DbLiteracyMaterial } from '../../literacy-material/model';
+import { DbLocation } from '../../location/model';
+import { DbOrganization } from '../../organization/model';
+import { DbPartner } from '../../partner/model';
+import { DbPartnership } from '../../partnership/model';
+import { DbPeriodicReport } from '../../periodic-report/model';
+import { DbPost } from '../../post/model';
+import { StepProgress } from '../../product-progress/dto';
+import { DbProduct } from '../../product/model';
+import { DbProjectChangeRequest } from '../../project-change-request/model';
+/* eslint-disable @typescript-eslint/naming-convention */
+import { DbProject } from '../../project/model';
+import { DbProjectMember } from '../../project/project-member/model';
+import { DbSong } from '../../song/model';
+import { DbStory } from '../../story/model';
+import { DbEducation, DbUnavailability, DbUser } from '../../user/model';
+import { Role } from '../dto';
+import { Powers } from '../dto/powers';
+import { DbBaseNodeGrant, DbRole } from '../model';
+
+// do not auto format this file
+// turned off prettier for role files to prevent auto-format making this file huge
+
+const read = true;
+const write = true;
+
+export const ExperienceOperations = new DbRole({
+  name: `global:${Role.ExperienceOperations}` as const,
+  powers: [
+    Powers.CreateEducation,
+    Powers.CreateProjectMember,
```

Comment:
In looking at the spreadsheet Seth attached to the Issue, I'm unsure if the CreateProjectMember power should be allowed?